Formula/python-tabulate.rb

```diff
@@ -16,10 +17,12 @@ class PythonTabulate < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f5bd465f5fd9a0e3eb107c4cdd244b63090a2ce102346f7d7fecfcdbabc1812a"
   end
 
-  depends_on "python@3.9"
+  depends_on "python@3.10"
 
   def install
-    system Formula["python@3.9"].bin/"python3", *Language::Python.setup_install_args(prefix)
+    python = Formula["python@3.10"].bin/"python3"
+    system python, *Language::Python.setup_install_args(prefix),
+           "--install-lib=#{prefix/Language::Python.site_packages(python)}"
```

Comment:
Alternatively, remove `keg_only` from `libpython-tabulate`, add it as a dependency, and then just install the Python script via something like:
```suggestion
   system "python3", "setup.py", "--no-user-cfg", "install_scripts", "--install-dir=#{bin}", "--skip-build"
```

Or some other variation based on `Language::Python.setup_install_args`.

This will only install the script using Python@3.10. The functionality will depend on library linked in `libpython-tabulate`.

Reply:
Done